### DISCUSSION TITLE
Play services updating message

### DISCUSSION THREAD

**Discussion by hawajka76** [Genymobile/feedback#63] [Genymotion SaaS] [answered]
Apologies if I missed the answer to this somehow, but I could not find it!
I've installed an application from the play store via opengapps. It loads, however the map interface does not. Instead it shows a white background with the following message: Play services updating. The map never appears. Is there a way to get it to work? See also the attached image. 
![Services updating](https://github.com/Genymobile/feedback/assets/35002520/5cb1495e-d2f5-40ba-b5e9-8e0be06fd9de)

**Comment by guillaume-roche** [comment, marked as answer]
Hello @hawajka76 

This discussion board is not suitable for support queries. Please contact our support team via [our help center](https://support.genymotion.com/hc/en-us).

Thank you for your understanding.
